src/components/Heading/Heading.test.js

```diff
@@ -1,72 +1,183 @@
 import Heading from '@components/Heading'
 
 const props = {
   children: 'Heading placeholder',
   level: 1
 }
 
-const headingComponent = <Heading {...props}>{props.children}</Heading>
-
 describe('<Heading/> mount', () => {
-  let wrapper
-
-  beforeEach(() => {
-    wrapper = mount(headingComponent)
-  })
-
-  afterEach(() => {
-    wrapper.unmount()
-  })
-
   it('is mounted', () => {
-    expect(wrapper.exists()).toBe(true)
+    render(<Heading {...props}>{props.children}</Heading>)
   })
 
   it('has `heading` class', () => {
-    expect(wrapper.find('span').hasClass('heading')).toBe(true)
+    render(<Heading {...props}>{props.children}</Heading>)
+
+    const heading = screen.getByTestId('heading')
+
+    expect(heading).toHaveClass('heading')
   })
 
   it('when prop `level 1`, has `h1` class', () => {
-    wrapper.setProps({
-      level: 1
-    })
-    expect(wrapper.find('span').hasClass('h1')).toBe(true)
+    render(<Heading {...props}>{props.children}</Heading>)
+
+    const heading = screen.getByTestId('heading')
+
+    expect(heading).toHaveClass('h1')
   })
 
   it('when prop `level 2`, has `h2` class', () => {
-    wrapper.setProps({
-      level: 2
-    })
-    expect(wrapper.find('span').hasClass('h2')).toBe(true)
+    render(
+      <Heading {...props} level={2}>
+        {props.children}
+      </Heading>
+    )
+
+    const heading = screen.getByTestId('heading')
+
+    expect(heading).toHaveClass('h2')
   })
 
   it('when prop `level 3`, has `h3` class', () => {
-    wrapper.setProps({
-      level: 3
-    })
-    expect(wrapper.find('span').hasClass('h3')).toBe(true)
+    render(
+      <Heading {...props} level={3}>
+        {props.children}
+      </Heading>
+    )
+
+    const heading = screen.getByTestId('heading')
+
+    expect(heading).toHaveClass('h3')
   })
 
   it('when prop `level 4`, has `h4` class', () => {
-    wrapper.setProps({
-      level: 4
-    })
-    expect(wrapper.find('span').hasClass('h4')).toBe(true)
+    render(
+      <Heading {...props} level={4}>
+        {props.children}
+      </Heading>
+    )
+
+    const heading = screen.getByTestId('heading')
+
+    expect(heading).toHaveClass('h4')
   })
 
   it('when prop `level 5`, has `h5` class', () => {
-    wrapper.setProps({
-      level: 5
-    })
-    expect(wrapper.find('span').hasClass('h5')).toBe(true)
+    render(
+      <Heading {...props} level={5}>
+        {props.children}
+      </Heading>
+    )
+
+    const heading = screen.getByTestId('heading')
+
+    expect(heading).toHaveClass('h5')
   })
 
   it('when prop `level 6`, has `h6` class', () => {
-    wrapper.setProps({
-      level: 6
-    })
-    expect(wrapper.find('span').hasClass('h6')).toBe(true)
+    render(
+      <Heading {...props} level={6}>
+        {props.children}
+      </Heading>
+    )
+
+    const heading = screen.getByTestId('heading')
+
+    expect(heading).toHaveClass('h6')
+  })
+
+  it('when prop `bold` is `false`, has `heading--bold` class', () => {
```

Comment:
test z lini 92 i 100 sprawdza to samo w sensie jest błąd logiczny w jednym na bank powinno być, zaprzeczenie w opisie tu has a tu hasn't

Reply:
Leci poprawka.